bridget/eddy/footprint.py

```diff
@@ -15,13 +15,22 @@
 
 """
 
-def footprint():
+def footprint(height = 2, t_air, a, p, wndspeed, cov_uw, cov_vw, var_v, method = "Kormann_Meixner"):
     """
-    Top-level footprint function. 
+    Top-level footprint function.
 
-    If more than one footprint method is implemented, these should go 
-    into functions in this module. This function then has a **string** 
+    If more than one footprint method is implemented, these should go
+    into functions in this module. This function then has a **string**
     parameter to switch the method.
-    
+
+    Parameters:
+        - height = measurement height (z) [m]
+        - wndspeed = horizontal wind component for the direction in which the sonic is oriented (u) [m/s]
```

Comment:
The data has `u, v, w` as separated columns, if we only need u, we could also rename this to `u`.